### PR TITLE
MODKBEKBJ-238 - Create periodic task to populate holdings table

### PR DESCRIPTION
**Pull request by natali-zaitseva**
## Purpose
As a part of https://issues.folio.org/browse/MODKBEKBJ-238 we need to be able to acquire information about staging snapshot. 

## Approach
* created endpoints to communicate with a staging server
* added tests

### CODE REVIEW

File: src/main/java/org/folio/holdingsiq/service/impl/HoldingsRequestHelper.java [hunk 1]
```diff
@@ -114,6 +114,24 @@ public HoldingsRequestHelper(Configuration config, Vertx vertx) {
     return future;
   }
 
+  protected CompletableFuture<Void> postHoldingsRequest(String query){
```
Comment (dmtkachenko) [inline src/main/java/org/folio/holdingsiq/service/impl/HoldingsRequestHelper.java]: Is this a general purpose method?
If not don't you think it should be next to its usage

File: src/main/java/org/folio/holdingsiq/service/LoadService.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import org.folio.holdingsiq.model.Holdings;
+import org.folio.holdingsiq.model.HoldingsLoadStatus;
+
+public interface LoadService {
+
+  /**
+   * Populates holdings data to a staging area
+   */
+  CompletableFuture<Void> populateHoldings();
```
Comment (dmtkachenko) [inline src/main/java/org/folio/holdingsiq/service/LoadService.java]: Could you please let me know more details about the purpose of this method

Comment (natali-zaitseva) [reply src/main/java/org/folio/holdingsiq/service/LoadService.java]: according to the doc - https://developer.ebsco.com/interactive/holdingsiq there three steps needed to be performed :
1 - POST holdings to the stage area
2 - GET status of the snapshot
3 - GET holdings if status from the previous step is "Completed"

Comment (dmtkachenko) [reply src/main/java/org/folio/holdingsiq/service/LoadService.java]: thanks, got it

**Pull request merged**